docs/influxdb.md

```diff
@@ -19,7 +19,7 @@ $ kubectl create -f deploy/kube-config/influxdb/
 Grafana service by default requests for a LoadBalancer. If that is not available in your cluster, consider changing that to NodePort. Use the external IP assigned to the Grafana service,
 to access Grafana.
 The default user name and password is 'admin'.
-Once you login to Grafana, add a datasource that is InfluxDB. The URL for InfluxDB will be `http://localhost:8086`. Database name is 'k8s'. Default user name and password is 'root'.
+Once you login to Grafana, add a datasource that is InfluxDB. The URL for InfluxDB will be `http://monitoring-influxdb:8086`. Database name is 'k8s'. Default user name and password is 'root'.
```

Comment:
I proposed to use monitoring-influxdb because it's reflecting whats used in the deployment.
But I agree http://INFLUXDB_HOST:INFLUXDB_PORT is better for understanding.